usaon_vta_survey/templates/users.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+  <h2>{% block title %}Users{% endblock %}</h1>
+
+  {% if not current_user.role=='admin' %}
```

Comment:
Does this work as expected for logged out users? I'd expect an error like "undefined has no attribute 'role'"

Reply:
Updated this to reflect that.